src/main.ts

```diff
@@ -141,13 +143,14 @@ export default class LatexSuitePlugin extends Plugin {
 
 		this.showSnippetsLoadedNotice(snippets.length, Object.keys(snippetVariables).length,  becauseFileLocationUpdated, becauseFileUpdated);
 
-		return sortSnippets(snippets);
+		return snippets;
 	}
 
 	async processSettings(becauseFileLocationUpdated = false, becauseFileUpdated = false) {
 		this.CMSettings = processLatexSuiteSettings(await this.getSnippets(becauseFileLocationUpdated, becauseFileUpdated), this.settings);
 		this.reconfigureLatexSuiteConfig();
 		this.refreshCMExtensions();
+		this.app.workspace.updateOptions();
```

Comment:
Moved this line outside of `refreshCMExtensions` to clarify the function's role.

Reply:
Maybe it is better to rename `refreshCMExtensions`.